ssh/util/uname.lua

```diff
@@ -0,0 +1,40 @@
+local uname = {}
+
+function uname.getResponse(message)
+    local unameCommand = {
+        { -- busybox_w32_32x
+            s = "Windows_NT",
```

Comment:
I would use the newly added ab-tester for this. Store this data concatened with a random key in the ab-tester, fetch a random one and split it again